### PR TITLE
fix: update dashboard, etc, ingresss versions, fix broken features af…

### PR DESCRIPTION
**Pull request by ikatlinsky**
This pull request addresses issues after merge of v2 chart from dev to master branch:

1. Versions of etc, ingress controller and dashboard are outdated
2. Some features that worked in v1 are now broken in v2 (apisix annotatation templates, custom secret keys for admin credentials)

Potentially there can be more issues for ones who will migrate from v1 to v2 version of the chart.

As a result of changes version of the apisix chart is updated to 2.7.0.

### CODE REVIEW

**Comment by ikatlinsky** [comment]
@AlinsRan @Revolyssup @moonming please review and accept

**Comment by ikatlinsky** [comment]
@AlinsRan @Revolyssup @moonming merged with 2.7.0 version of the chart and updated the version in this PR to 2.8.0

**Comment by ikatlinsky** [comment]
@AlinsRan @Revolyssup @moonming up! thanks and sorry for pinging

**Comment by ikatlinsky** [comment]
@AlinsRan @Revolyssup @moonming up

**Review by AlinsRan — APPROVED** [review]

**Comment by ikatlinsky** [comment]
@Revolyssup @moonming up

File: charts/apisix/templates/_helpers.tpl [hunk 1]
```diff
@@ -135,3 +135,25 @@ Return the password key name of etcd secret
...
+{{/*
+Key to use to fetch viewer token from secret
+*/}}
+{{- define "apisix.admin.credentials.secretViewerKey" -}}
+{{- if .Values.admin.credentials.secretViewerKey }}
+{{- .Values.admin.credentials.secretViewerKey }}
+{{- else }}
+{{- "viewer" }}
+{{- end }}
+{{- end }}
```
Comment (Gallardot) [inline charts/apisix/templates/_helpers.tpl:159]: New line missing end of file. Rest LGTM

**Review by Gallardot — CHANGES_REQUESTED** [review]

File: charts/apisix/templates/_helpers.tpl (hunk 1 above)
Comment (ikatlinsky) [reply charts/apisix/templates/_helpers.tpl:159]: thanks, fixed

Comment (ikatlinsky) [reply charts/apisix/templates/_helpers.tpl]: @Gallardot please, recheck

**Review by Gallardot — APPROVED** [review]

**Comment by ikatlinsky** [comment]
@Gallardot @AlinsRan hi, any chances that this pr can be merged? thanks

**Pull request merged**